### PR TITLE
refactor: refactored code and added theme

### PR DESCRIPTION
**Pull request by krate27**
Added dark theme and did some refactoring!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,8 +13,12 @@ import { SinglePlay } from "./Pages/SinglePlaylistNav/singlePlay";
 import { VideoListing } from "./Pages/Video-Listing/videoListing";
 import { WatchLater } from "./Pages/WatchLater/watchLater";
 function App() {
+  const {theme} = useTheme();
   return (
-    <div className="App">
+    <div className="App" style={{
+      color: theme === "light" ? "black" : "#fff",
+      backgroundColor: theme === "light" ? "#e0e0eb" : "#141319",
```
Comment (abhij1607) [inline src/App.js:20]: suggestion to use color variables

**Review by abhij1607 — APPROVED** [review]
Good folder structure
Neat and clean code
Good work with theme
Good to merge

Small Suggestion:
- toggle class name instead of using inline style

File: src/Pages/Liked-Videos/likeVideo.js [hunk 2]
```diff
@@ -23,7 +25,18 @@ const LikedVideo = () => {
...
+                    class="menu-card"
+                    style={{
+                      color: theme === "light" ? "black" : "#fff",
+                      backgroundColor:
+                        theme === "light" ? "#e0e0eb" : "#191921",
+                      boxShadow:
+                        theme === "light"
+                          ? "rgba(0, 0, 0, 0.4) 0px 2px 4px,rgba(0, 0, 0, 0.3) 0px 7px 13px -3px, rgba(0, 0, 0, 0.2) 0px -3px 0px inset"
+                          : "rgba(179, 179, 0) 5px 5px",
+                    }}
```
Comment (MaruthiPothuganti) [inline src/Pages/Liked-Videos/likeVideo.js:38]: A suggestion: Insteading of passing it everywhere, this can be made into class and used everywhere.

**Review by MaruthiPothuganti — COMMENTED** [review]
Made a suggestion and It would be good to have a live preview. Apart from that, Code refactoring done good. PR looks Good to merge.

**Pull request merged**